lib/suspenders/generators/app_generator.rb

```diff
@@ -52,10 +55,12 @@ def suspenders_customization
       invoke :setup_segment
       invoke :setup_bundler_audit
       invoke :setup_spring
+      invoke :generate_all
```

Comment:
It's more accurate for the future if we call this `#generate_default` instead of `#generate_all`.